src/main/java/io/temporal/samples/dsl/models/ActivityInvocation.java

```diff
@@ -36,7 +36,9 @@ public Void execute(Map<String, String> bindings) {
                 .setTaskQueue("dsl")
                 .build());
 
-    String results = stub.execute(this.method, String.class, new Object[] {args});
+    String methodNameDescriptor =
+        this.name + this.method.substring(0, 1).toUpperCase() + this.method.substring(1);
```

Comment:
I would use a single "activityType" string. Activities can be implemented in different languages like Go which have completely different approach to activity naming.